pkgs/top-level/python-packages.nix

```diff
@@ -18440,4 +18440,24 @@ let
     };
   };
 
+  importmagic = buildPythonPackage rec {
+    simpleName = "importmagic";
+    name = "${simpleName}-${version}";
+    version = "0.1.3";
+    doCheck = false;  # missing json file from tarball
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/i/${simpleName}/${name}.tar.gz";
+      sha256 = "194bl8l8sc2ibwi6g5kz6xydkbngdqpaj6r2gcsaw1fc73iswwrj";
+    };
+
+    propagatedBuildInputs = with self; [ six ];
```

Comment:
I'm no python expert and not sure of this.
I i saw that other python packages declared it.
And, also, in the importmagic repository exists a `six.py` file and this seems related to retro-compatibility with python2.

So I added it.